.circleci/config.yml

```diff
@@ -32,12 +32,13 @@ jobs:
           command: |
             python3 -m venv venv
```

Comment:
Don't really need venv anymore since pipenv creates a virtualenv also.

Reply:
I was mostly just focusing on requirements.txt in the first pass to use pipenv. commit 2ca1926 now contains the latest.